packages/react-components/src/autocomplete/src/Autocomplete.tsx

```diff
@@ -245,16 +264,20 @@ export function InnerAutocomplete(props) {
         // Usually provided by the field inputs.
         "aria-labelledby": ariaLabelledBy,
         "aria-describedby": ariaDescribedBy,
-        menuProps: { id: menuId, style: { width: menuWidth, ...menuStyle } = {}, ...menuProps } = {},
```

Comment:
This is the only hack i've done. I can't find a way to deconstruct the object inline there. I'll look into it tomorrow